spacelift/data_tool_versions.go

```diff
@@ -0,0 +1,95 @@
+package spacelift
+
+import (
+	"context"
+	"slices"
+
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+)
+
+func dataToolVersions() *schema.Resource {
+	return &schema.Resource{
+		Description: "Lists supported versions for a given tool.",
+		ReadContext: dataToolVersionsRead,
+		Schema: map[string]*schema.Schema{
+			"tool": {
+				Type:             schema.TypeString,
+				Description:      "The tool to get supported versions for.",
```

Comment:
Can we add the allowed values to the description, similarly to this?
https://github.com/spacelift-io/terraform-provider-spacelift/blob/c8054ad516c16b45d6feeb5dae18c4961029c104/spacelift/resource_stack.go#L579

Reply:
Done :+1: